### ISSUE TITLE
Showing Wrong RowSideColors instead of the specified ones

### ISSUE THREAD

**Issue by rezaeir** [talgalili/heatmaply#247]
# Describe the bug
I'm trying to reproduce the same plot generated from the following heatmap.2 code using heatmaply:
```
heatmap.2(diffGenes, 
          Rowv=as.dendrogram(clustRows), 
          RowSideColors=module.color,
          col=myheatcolors, scale='row', labRow=NA,
          density.info="none", trace="none",  
          cexRow=1, cexCol=1, margins=c(8,20))
```
The following is my code for heatmaply:
```
heatmaply(diffGenes,
          colors = myheatcolors,
          Rowv=as.dendrogram(clustRows),
          RowSideColors=module.color,
          showticklabels=c(TRUE,FALSE),
          scale='row')
```

# Expected behavior
I expect to get the right colors as specified exactly like what heatmap.2 gives me. I searched the hex code of colors and the colors are what heatmap.2 produces not what heatmaply gives me.
It doesn't relate to specifying several colors for clusters. I also checked this behavior on one cluster with all rows having the same colors and got one unrelated color instead of the actual specified color.

Specified Colors hex code:
"#FF9900", "#00FF33", "#0033FF", "#FF0099"

heatmap.2 RowSideColors:
![heatmap 2_colors](https://user-images.githubusercontent.com/47621967/88921023-0da9c500-d283-11ea-903e-26a2c64ca7fe.jpeg)

heatmaply RowSideColors:
![heatmaply](https://user-images.githubusercontent.com/47621967/88920989-02ef3000-d283-11ea-9448-dda8225eb209.jpeg)

# Version info
heatmaply version: '1.1.0'
R version: '4.0.2'

**Comment by Alanocallaghan** [comment]
When opening an issue please include a [reproducible example](https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example)

**Comment by Alanocallaghan** [comment]
Have you checked the current master because I think this is fixed https://github.com/talgalili/heatmaply/issues/246

**Comment by rezaeir** [comment]
> When opening an issue please include a [reproducible example](https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example)

Sorry about that. It is part of a large pipeline and hard to make a separate reproducible example from it.

**Comment by rezaeir** [comment]
> Have you checked the current master because I think this is fixed #246

Isn't it still accessible from the CRAN? Should I install it from Github?

**Comment by Alanocallaghan** [comment]
master has a bugfix for this issue that's not been pushed to cran yet.
